common/scripts/build-grub.sh

```diff
@@ -49,10 +49,10 @@ if [ $BAND == "SR" ]; then
 else
     . $TOP_DIR/../../common/config/common_config.cfg
 fi
+# toolchain
+. $TOP_DIR/../../common/scripts/common_cross_toolchain.sh
```

Comment:
Please refactor the changes in file to a new file common/scripts/build-grub-arm.sh

The original file common/scripts/build-grub.sh shall be unmodified.